helm/chaos-mesh/values.yaml

```diff
@@ -70,6 +70,8 @@ chaosDaemon:
   # runtime: containerd
   # socketPath: /run/containerd/containerd.sock
 
+  tolerations: []
```

Comment:
```
      - key: node-role.kubernetes.io/master
        effect: NoSchedule
``` 

How about setting a default value?

Reply:
We have never observed any problems with master nodes (chaos-daemon pod created well on them) but yes we can add a default value.

Reply:
@cwen0 I prefer not to set default value actually. 

Because we have no place to tell/warn user that our chaos-daemon won't follow your guide to place pods.